database/config.py

```diff
@@ -1,3 +1,3 @@
 ### config.py ###
 
-DATABASE_URI = "postgres+psycopg2://USERNAME:PASSWORD@IP_ADDRESS:0/DATABASE_NAME"
\ No newline at end of file
+DATABASE_URI = "postgres+psycopg2://postgres:bentopostgres@localhost:5432/test3"
```

Comment:
don't commit this btw (you can uncheck the config file before comitting your changes)

Reply:
oo right yep sure thing